docs/getting_started/quick_start/fill_box_example.rst

```diff
@@ -0,0 +1,78 @@
+Fill Box
+========
+
+
+All-Atom (AA) Hexane and Ethanol System
+---------------------------------------
+
+**Note**: `foyer <https://foyer.mosdef.org/en/stable/>`_ is used in conjunction with **mBuild** in the following example to demonstrate how all the `MoSDeF <https://mosdef.org>`_ software integrates to generate a simulation box.
+
+Import the required mbuild package.
+
+.. code:: ipython3
+
+    import mbuild as mb
+
+
+Construct an all-atom (AA) hexane and ethanol using the OPLS-AA force field (FF),
+which is comes as a standard `foyer <https://foyer.mosdef.org/en/stable/>`_ force field (FF).
```

Comment:
(FF) is defined here twice. Line 17 and line 18. Probably only need to do it once.

Reply:
FF was removed from the last line